date-input.html

```diff
@@ -69,6 +70,7 @@
           readonly$="[[readonly]]">
       <span>/</span>
       <input is="iron-input"
+          id="expYear"
```

Comment:
Agreed. I was going by the autocomplete="cc-exp-year". I'll change it to expirationYear. Seems a bit more descriptive the dateYear.